Automattic/jetpack
# Adds Javascript translation files to jetpack-mu-wpcom

Fixes Automattic/i18n-issues#858 and https://github.com/Automattic/wp-calypso/issues/94070 

## Proposed changes:

* Adds JavaScript translation files to ensure that the `Welcome Guide` and `Limited Global Styles` and `Page Patterns` selector are fully translated.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Sandbox a simple site
* Execute `bin/jetpack-downloader test jetpack fix/etk-translation-issues && bin/jetpack-downloader test jetpack-mu-wpcom-plugin fix/etk-translation-issues` to release these changes on the sandbox
* Verify that the Global Styles, Welcome Guide, and Page Patterns selector are localized
